examples/example_array_as_a_sensor.py

```diff
@@ -0,0 +1,119 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from kwave.data import Vector
+from kwave.kgrid import kWaveGrid
+from kwave.kmedium import kWaveMedium
+from kwave.ksensor import kSensor
+from kwave.ksource import kSource
+from kwave.kspaceFirstOrder2D import kspace_first_order_2d_gpu
+from kwave.options.simulation_execution_options import SimulationExecutionOptions
+from kwave.options.simulation_options import SimulationOptions
+from kwave.utils.conversion import cart2grid
+from kwave.utils.kwave_array import kWaveArray
+from kwave.utils.mapgen import make_cart_circle, make_disc
+from kwave.utils.signals import reorder_binary_sensor_data
+
+
+def main():
+    # create empty array
+    karray = kWaveArray()
+
+    # define arc properties
+    radius = 100e-3  # [m]
+    diameter = 8e-3  # [m]
+    ring_radius = 50e-3  # [m]
+    num_elements = 20
+
+    # orient all elements towards the center of the grid
+    focus_pos = Vector([0, 0])  # [m]
+
+    element_pos = make_cart_circle(ring_radius, num_elements, focus_pos)
+
+    for idx in range(num_elements):
+        karray.add_arc_element(element_pos[:, idx], radius, diameter, focus_pos)
+
+    # grid properties
+    Nx = 256
+    dx = 0.5e-3
+    Ny = 256
+    dy = 0.5e-3
```

Comment:
Perhaps use `Vector` here to define variables?

Reply:
Done

Reply:
Done